apigw-lambda-cognito-sam-java/template.yaml

```diff
@@ -0,0 +1,138 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Description: This template exposes API to create and confirm user in Amazon Cognito user pool
+
+Globals:
+  Function:
+    Runtime: java21
+    MemorySize: 3008
+    Timeout: 120
```

Comment:
Is that necessary with an API Gateway timeout of 29 seconds?

Reply:
Fixed.